src/registry_schemas/example_data/schema_data.py

```diff
@@ -1000,6 +1000,87 @@
     }
 }
 
+COOP_INCORPORATION = {
+    'nameRequest': {
+        'legalType': 'CP'
+    },
+    'nameTranslations': [
+        {'name': 'ABC Ltd.'},
+        {'name': 'Financière de l’Odet'},
+        {'name': 'Société Générale'}
+    ],
+    'offices': {
+        'registeredOffice': {
+            'deliveryAddress': {
+                'streetAddress': 'delivery_address - address line one',
+                'addressCity': 'delivery_address city',
+                'addressCountry': 'CA',
+                'postalCode': 'H0H0H0',
+                'addressRegion': 'BC'
+            },
+            'mailingAddress': {
+                'streetAddress': 'mailing_address - address line one',
+                'addressCity': 'mailing_address city',
+                'addressCountry': 'CA',
+                'postalCode': 'H0H0H0',
+                'addressRegion': 'BC',
+            }
+        }
+    },
+    'parties': [
+        {
+            'officer': {
+                'id': 1,
+                'firstName': 'Joe',
+                'lastName': 'Swanson',
+                'middleName': 'P',
+                'email': 'joe@email.com',
+                'orgName': '',
+                'partyType': 'person'
+            },
+            'mailingAddress': {
+                'streetAddress': 'mailing_address - address line one',
+                'streetAddressAdditional': '',
+                'addressCity': 'mailing_address city',
+                'addressCountry': 'CA',
+                'postalCode': 'H0H0H0',
+                'addressRegion': 'BC'
+            },
+            'deliveryAddress': {
+                'streetAddress': 'delivery_address - address line one',
+                'streetAddressAdditional': '',
+                'addressCity': 'delivery_address city',
+                'addressCountry': 'CA',
+                'postalCode': 'H0H0H0',
+                'addressRegion': 'BC'
+            },
+            'roles': [
+                {
+                    'roleType': 'Completing Party',
+                    'appointmentDate': '2018-01-01'
+
+                },
+                {
+                    'roleType': 'Director',
+                    'appointmentDate': '2018-01-01'
+
+                }
+            ]
+        }
+    ],
+    'contactPoint': {
+        'email': 'no_one@never.get',
+        'phone': '123-456-7890'
+    },
+    'cooperative': {
```

Comment:
I wonder if we can start making the examples fit the same as the schema definition?
a value for _ cooperative_ that is then added to the base incorporation?

Reply:
Do you suggest to include _cooperative_ in incorporation (example) itself?

Reply:
in the default incorporation (core) no.
I think we should split these out so that we end up with _core_, and then IA-, ben, ltd, coop, etc.